nixos/doc/manual/configuration/customizing-packages.xml

```diff
@@ -17,7 +17,10 @@
  <warning>
   <para>
    Unfortunately, Nixpkgs currently lacks a way to query available
-   configuration options.
+   configuration options from command line. You can however go to
```

Comment:
These are different options than the section is talking about.

You are describing NixOS module options but this is talking about package options (that is the attribute set you pass as `config` when creating `nixpkgs`, or through [`nixpkgs.config` NixOS option](https://nixos.org/nixos/options.html#nixpkgs.config). There still is not a way to query those as far as I am aware.

And for NixOS options, you can actually use `man configuration.nix` or `nixos-option` commands.